src/Tools/Common/DefaultConsole.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Diagnostics.Tools.Common
     /// maybe we could map it to System.CommandLine's IConsole, but right now that interface doesn't
     /// have enough functionality for everything we need.
     /// </summary>
-    internal class DefaultConsole : IConsole
+    internal sealed class DefaultConsole : IConsole
     {
         private readonly bool _useAnsi;
         public DefaultConsole(bool useAnsi)
```

Comment:
```suggestion
        public DefaultConsole(bool useAnsi = false)
```

If we are going to be using the DefaultConsole repeatedly and always passing false we may as well make it the default.

Reply:
I defaulted CollectLinux and Collect's to use false in prior PRs, but I'm not sure if they should try to emit ansi. Changed the default for DefaultConsole as false